scripts/process_licenses.py

```diff
@@ -61,6 +61,64 @@
     "org.bouncycastle:bcpkix-jdk18on": BOUNCY_CASTLE_LICENSE,
     "org.bouncycastle:bcutil-jdk18on": BOUNCY_CASTLE_LICENSE,
     "org.bouncycastle:bcprov-jdk18on": BOUNCY_CASTLE_LICENSE,
+    "com.amazonaws:aws-java-sdk-core": APACHE_LICENSE,
```

Comment:
Did you check all of these don't ship with a license?

Reply:
Yes, I checked the artifact directory to see if there's a license file. If not, I used the output from `license:add-third-party` directly.